src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -29,6 +29,7 @@ import com.github.dockerjava.api.model.RestartPolicy
 import com.github.dockerjava.api.model.Volume
 import com.github.dockerjava.api.model.VolumesFrom
 import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.MapProperty
```

Comment:
The use of `MapProperty` raises the minimum compatible Gradle version to 5.1. Please update the note in the user guide to reflect that.